CMakeLists.txt

```diff
@@ -57,6 +57,13 @@ endif()
 if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   # using regular Clang or AppleClang
   set(LLVM_CLANG true)
+  if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "10.0.0" AND
+     CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    message(WARNING "Found LLVM < 10 which has known optimization bugs with \
```

Comment:
Please name the glibc version. The issue starts with glibc-2.31
https://github.com/bminor/glibc/commit/7bdb921d70bf9f93948e2e311fef9ef439314e41

On Ubuntu we have not issue because Ubuntu Bionic uses glibc-2.27 and LVVM 9 
Ubuntu Focal has 2.31 but also LVVM 10.